src/aks-preview/HISTORY.md

```diff
@@ -6,6 +6,7 @@ Release History
 0.5.67
 ++++++
 * Add support for csi drivers extensibility.
+* Add support for apiserver vnet integration.
```

Comment:
version is not bumped for last pr `csi drivers extensibility`, so I added this feature in 0.5.67